_pages/index.md

```diff
@@ -41,8 +41,18 @@ Le bureau de l'association est actuellement composé de :
 - Lean Camp
 
 
+# Assemblées générales
+
+Les assemblées générales sont les réunions annuelles lors desquelles le bureau présente son bilan, où le bureau peut être renouvelé, et où de grandes orientations peuvent être décidées.
+
+{% assign assemblees_generales = site.assemblee_generale | reverse %}
+
+{% for ag in assemblees_generales %}
```

Comment:
Using `{% for ag in assemblees_generales -%}` looks to fix the thing but I'm not familiar with Jekyll so I let you amend if pertinent.

Reply:
The templating language used by Jekyll is Liquid, and you can find its doc at https://shopify.github.io/liquid/basics/whitespace/ 🙂

I think it wrapped each list item in a `<p>` because we had an empty line inbetween each list item in Markdown. Adding a final dash strips the trailing whitespace, leaving just one newline, making it an actual Markdown list.